src/lib.rs

```diff
@@ -72,6 +72,9 @@ mod sys;
 #[path="windows.rs"]
 mod sys;
 
+#[cfg(not(windows))]
+pub use sys::SharedChildExt;
```

Comment:
Should we figure out a way to get `unix` in the module path here? That might be weird given that we already have unix.rs, but we could rename that one to unix_sys.rs or something if we want?